apps/marketplace/components/Insights/insights.scss

```diff
@@ -0,0 +1,96 @@
+.blue-box {
+    background-color: rgba(167, 219, 251, 0.8);
+    border-color: #666666;
+    border-width: 2px;
+    border-style: solid;
+    text-align: center;
+    height: 200px;
+    width: 100%;
+    margin: 1rem 0rem;
+    padding: 2rem;
+}
+.blue-box > .big-text {
+    font-size: 2rem;
+    font-weight: bold;
+    color: #0985D1;
+}
+
+.chart-md-height-1x {
+    position: relative;
+    height: 12rem;
+}
+
+.chart-md-height-2x {
+    position: relative;
+    height: 14rem;
+}
+
+.chart-md-height-3x {
```

Comment:
This style and some others aren't used.  Do we need them?

Reply:
added so we don't have gaps